client/src/views/WorkspaceDetail.vue

```diff
@@ -122,39 +172,92 @@
+            <v-spacer />
+            <v-dialog
+              v-model="graphDialog"
+              width="700"
+            >
+              <template v-slot:activator="{ on }">
+                <v-btn
+                  class="pl-3 pr-2"
+                  color="blue darken-2"
+                  flat
+                  v-on="on"
+                >
+                  New Graph
+                  <v-icon
+                    right
+                    size="20px"
+                  >add_circle</v-icon>
+                </v-btn>
+              </template>
+              <v-card>
+                <v-card>
+                  <v-card-title
+                    class="headline pb-0 pt-3"
+                    primary-title
+                  >
+                    Create Graph
+                  </v-card-title>
+
+                  <v-card-text class="px-4 pt-4 pb-1">
+                    <v-layout row wrap>
+                      <v-flex>
+                        <v-text-field
+                          box
+                          label="Graph name"
+                          v-model="newGraph"
+                          :error-messages="graphCreationErrors"
+                        />
+                      </v-flex>
+                    </v-layout>
+
+                    <v-layout row wrap>
+                      <v-flex>
+                        <v-select
+                          box
+                          chips
+                          class="choose-tables"
+                          clearable
+                          deletable-chips
+                          label="Choose node tables"
+                          multiple
+                          v-model="graphNodeTables"
+                          :items="nodeTables"
+                        />
+                      </v-flex>
+                    </v-layout>
+
+                    <v-layout row wrap>
+                      <v-flex>
+                        <v-select
+                          box
+                          label="Choose edge table"
+                          v-model="graphEdgeTable"
+                          :items="edgeTables"
+                        />
+                      </v-flex>
+                    </v-layout>
+                  </v-card-text>
+
+                  <v-divider></v-divider>
+
+                  <v-card-actions class="px-4 py-3">
+                    <v-spacer></v-spacer>
+                    <v-btn :disabled="graphCreateDisabled" @click="createGraph">create graph</v-btn>
+                  </v-card-actions>
+                </v-card>
+              </v-card>
+            </v-dialog>
+          </v-card-title>
+
+            <v-card-text class="pt-0">
+              <v-divider />
+              <div class="content pt-3">
+                <p class="grey--text text--darken-1">Graphs info Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus non est et nisl egestas molestie. Morbi congue varius ipsum.</p>
```

Comment:
How hard is it to move this inside a little question mark popover-type thing next to the "Tables" title?

I will also need to send you some copy for this (and the other lorem ipsum block).

Reply:
@waxlamp - it doesn't look great with the design, imo. I have a couple ideas if you want to discuss at my desk at some point (and I can show you what I'm talking about).

Reply:
Based on our conversation, we're disregarding this for now.